src/main/resources/application.properties

```diff
@@ -19,6 +19,18 @@ hocs.search-service=http://localhost:8088
 hocs.document-service=http://localhost:8083
 hocs.basicauth=UNSET
 
+bank-holidays-api-url=https://www.gov.uk/bank-holidays.json
```

Comment:
We will need to add this to the outbound proxy allow list
https://github.com/UKHomeOffice/kube-hocs/blob/main/kube/hocs-outbound-proxy-configmap.yaml#L21

Reply:
I've added this https://github.com/UKHomeOffice/kube-hocs/pull/89